src/main/java/edu/cornell/kfs/pmw/batch/service/impl/PaymentWorksNewVendorRequestsReportServiceImpl.java

```diff
@@ -24,71 +26,28 @@
-    private String taxIdTypeLabel = null;
-    private String vendorSubmitterEmailLabel = null;
-    private String initiatorNetidLabel = null;
-    private String errorsLabel = null;
-    
-    @Override
-    public PaymentWorksBatchReportVendorItem createBatchReportVendorItem(PaymentWorksVendor pmwVendorProcessed) {
-        return new PaymentWorksBatchReportVendorItem(pmwVendorProcessed.getPmwVendorRequestId(),
-                                                     pmwVendorProcessed.getProcessTimestamp(),
-                                                     pmwVendorProcessed.getVendorType(), 
-                                                     pmwVendorProcessed.getRequestingCompanyLegalName(),
-                                                     pmwVendorProcessed.getRequestingCompanyLegalFirstName(),
-                                                     pmwVendorProcessed.getRequestingCompanyLegalLastName(), 
-                                                     pmwVendorProcessed.getRequestingCompanyTinType(),
-                                                     pmwVendorProcessed.getRequestingCompanyCorporateEmail(),
-                                                     pmwVendorProcessed.getInitiatorNetId(),
-                                                     null);
-    }
-
-    @Override
-    public PaymentWorksBatchReportVendorItem createBatchReportVendorItem(PaymentWorksVendor pmwVendorWithErrors, List<String> errorMessages) {
-        return new PaymentWorksBatchReportVendorItem(pmwVendorWithErrors.getPmwVendorRequestId(),
-                                                     pmwVendorWithErrors.getProcessTimestamp(),
-                                                     pmwVendorWithErrors.getVendorType(), 
-                                                     pmwVendorWithErrors.getRequestingCompanyLegalName(),
-                                                     pmwVendorWithErrors.getRequestingCompanyLegalFirstName(),
-                                                     pmwVendorWithErrors.getRequestingCompanyLegalLastName(), 
-                                                     pmwVendorWithErrors.getRequestingCompanyTinType(),
-                                                     pmwVendorWithErrors.getRequestingCompanyCorporateEmail(),
-                                                     pmwVendorWithErrors.getInitiatorNetId(),
-                                                     errorMessages);
-    }
-
     @Override
     public void generateAndEmailProcessingReport(PaymentWorksNewVendorRequestsBatchReportData reportData) {
         LOG.debug("generateAndEmailReport entered");
         File reportFile = generateReport(reportData);
         sendResultsEmail(reportData, reportFile);
     }
     
+    public void sendEmailThatNoDataWasFoundToProcess(List<String> emailSubjectItems, List<String> emailBodyItems) {
+        super.sendEmailThatNoDataWasFoundToProcess(emailSubjectItems, emailBodyItems); 
+    }
```

Comment:
Here's another instance of an override that doesn't seem to be doing anything apart from calling the superclass method, where the superclass method already appears to have the same visibility. Sorry for not pointing it out sooner.